dd2480/README.md

```diff
@@ -281,6 +285,15 @@ There are many opportunities to split up the code into smaller functions. We wou
 7. Extract the month parsing logic
 8. Extract multi-predicate `else if` statements with methods  
 
+### FieldNameLabel::getDescription
+This function is used to get the description message of each fields.
+The decriptions of three fields are combined in this function.
+
+The refactoring plan is to split the methods into three parts for each field. 
+The decriptions for each field are indiviually stored in a new class. So in the target method,
+it would call other methods for specific field to get the descriptions which avoid lots of 
+switch statements and reduce complexity.
+
 ## Task 3 - Refactoring Implementation
```

Comment:
Could we also document the outcome of the refactoring for `FieldNameLabel`? Write down what CCN we started with and the reduction after refactoring